doc/configuration.md

```diff
@@ -335,6 +337,54 @@ proposals. Does not control `epoch_stake` and `rewards`, For this check `ledger`
 
  * Type: `boolean`
 
+##### Stake Addresses
+
+Whitelist stake addresses. Only set values will be kept in the database, all others will be ignored.
+
+`stake_addresses`
+ 
+ * Type: `string`
+
+The string will be validated and needs to omit `\x` from the start of the hash eg:
+
+`\x6c969320597b755454ff3653ad09725d590c570827a129aeb4385526`
+should be entered as:
+`6c969320597b755454ff3653ad09725d590c570827a129aeb4385526`
+
+Be mindfull that whitelisting stake address will omit data that does not match being present from the following tables.
+
++-----------------------+
+|      table name       |
+|-----------------------+
+| collateral_tx_out     |
+| delegation            |
+| delegation_vote       |
+| epoch_stake           |
+| gov_action_proposal   |
+| instant_reward        |
+| pool_owner            |
+| pool_update           |
+| reserve               |
+| stake_deregistration  |
+| stake_registration    |
+| treasury              |
+| treasury_withdrawal   |
+| tx_out                |
+| withdrawal            |
++-----------------------+
```

Comment:
Are there any queries done by db-sync to these table? Are they still done If the entries are no longer inserted? How does omiting these table affect the rest of db-sync

Reply:
@ArturWieczorek do you think the count of `tx_out` be the same as `collateral_tx_out`?

Reply:
@Cmdv No, my point is that there is no difference in the number of records for those tables between white-listed version of `preprod` DB and "regular" version (non white-listed) of `preprod` DB. Those tables contain `stake_address_id` field so I would expect that only records that match my stake address  would be insterted , not those with `NULL` `stake_address_id` .